arch/arc/include/asm/page.h

```diff
@@ -107,6 +114,17 @@ typedef struct page *pgtable_t;
 #define pfn_valid(pfn)		(((pfn) - ARCH_PFN_OFFSET) < max_mapnr)
 #endif
 
+#ifdef CONFIG_64BIT
+
+#define __pa(vaddr)  		((unsigned long)(vaddr) - \
+				PAGE_OFFSET + \
+				CONFIG_LINUX_LINK_BASE)
+#define __va(paddr)  		((void *)((unsigned long)(paddr) - \
+				CONFIG_LINUX_LINK_BASE + \
+				PAGE_OFFSET))
+
+#else /* V2 and Compact */
+
```

Comment:
IIUC this bit belongs to the previous commit from this MR.

Reply:
If I will move this change to the previous commit it may break things for case where CONFIG_LINUX_LINK_BASE != PAGE_OFFSET. Because before `ARC: mm: Move kernel to 0xffff_0000_0000_0000` we used LINK_BASE as PAGE_OFFSET in some places.